complete_CS_from_basic_to_advance.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
- Add proper title and description at the beginning of the file
- Name your file appropriately describing the content
- Please add at least 5 resources (you can add a new table below this one and suggest the top 5 courses provided by OSSU in your opinion)

Reply:
It's a great resource @charan-143 but please make above-mentioned changes